cvat/apps/engine/tests/test_rest_api_3D.py

```diff
@@ -761,5 +791,5 @@ def test_api_v2_export_dataset(self):
                             with open(file_name, "wb") as f:
                                 f.write(content.getvalue())
                         self.assertEqual(osp.exists(file_name), edata['file_exists'])
-                        self._check_dump_content(content, task_ann_prev.data, format_name,related_files=False)
+                        self._check_dump_content(content, task_ann_prev.data, format_name, related_files=True)
```

Comment:
```suggestion
                        if edata['file_exists']:
                            self._check_dump_content(content, task_ann_prev.data, format_name, related_files=True)
```

It seems there is an error in the test.